### PR TITLE
Destroy a minor account

### PR DESCRIPTION
**Pull request by antopalidi**
This PR adds the ability to remove a minor from the list. 

If the minor has never logged in, the user hard destroys.
If the user is logged in, the user will be softly destroyed, the minor's  account and minor data will be deleted.

### CODE REVIEW

File: app/commands/decidim/kids/destroy_minor_account.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      def initialize(minor_user, minor_account)
+        @minor_user = minor_user
+        @minor_account = minor_account
+      end
+
+      def call
+        transaction do
+          destroy_minor_user
+          destroy_minor_data
+          destroy_minor_account
```
Comment (microstudi) [inline app/commands/decidim/kids/destroy_minor_account.rb]:
```suggestion
          destroy_minor_data
          destroy_minor_account
          destroy_minor_user
```
Shoudn't we reverse the order? destroy the parent object last

**Review by microstudi — DISMISSED** [review]
Hi @antopalidi this looks good to me, the only think is that I'd destroy first the object depending on the user model and not the other way around.

File: app/commands/decidim/kids/destroy_minor_account.rb (hunk 1 above)
Comment (antopalidi) [reply app/commands/decidim/kids/destroy_minor_account.rb]: Of course. Corrected

**Review by fblupi — APPROVED** [review]
For me it's ok!

Thanks for your work 😄

**Pull request merged**